packages/RatPack.package/RPApplication.class/instance/setRouteHandlerFor..st

```diff
@@ -0,0 +1,7 @@
+routing
+setRouteHandlerFor: aHttpRequest
+	
+	| method methodRoutes |
+	method := aHttpRequest method asLowercase asSymbol asMutator.
+	methodRoutes := self routesFor: method.
+  	self routeHandler: (self findHandlerFor: aHttpRequest in: methodRoutes) value. 
```

Comment:
Because the findHandler Method (which wasn't written fro me, is completely bad.
It returns the the assosiation from the dictionary or aBlock ...

I changed it so the findHandler returns only the handlerSymbol

Also changed setRouteHandlerFor: ... to setRouteHandler